packages/language-server/src/completion/completions.ts

```diff
@@ -389,6 +442,56 @@ export function getValuesInsideBrackets(line: string): string[] {
   return result.map((v) => v.trim().replace('"', '').replace('"', ''))
 }
 
+
+async function declaredNativeTypes
+  (document: TextDocument): Promise<boolean> {
+  const binPath = await util.getBinPath()
+  let nativeTypes: NativeTypeConstructors[] = await nativeTypeConstructors(binPath, document.getText())
```

Comment:
Note to self: We check for the length of returned native types as opposed to the feature flag list. Which should be okay.